phoenix/src/madgwick_service.rs

```diff
@@ -0,0 +1,150 @@
+use madgwick::Marg;
+use messages::{Message, sensor::{self, SbgData, EkfQuat}};
+use messages::sensor::Sensor;
+use messages::sensor_status::EkfStatus;
+
+pub struct MadgwickService {
```

Comment:
Rust docs can be done using three slashes /// Without this notation they won't be built by cargo doc --open (I think). The 3 slashes should only be fro funcs, structs, enums, etc but general comments are still // in Rust. There are actually really neat features for Rust docs, like being able to run doc tests! https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html